lib/mongo/crypt/auto_encrypter.rb

```diff
@@ -117,7 +117,7 @@ def initialize(options)
           @crypt_handle.crypt_shared_lib_available? ||
           @options[:extra_options][:crypt_shared_lib_required]
 
-        if !@options[:extra_options][:crypt_shared_lib_required]
+        unless @options[:extra_options][:crypt_shared_lib_required] || @crypt_handle.crypt_shared_lib_available?
```

Comment:
Personally I only use `unless` if there is a single clause in the condition, i.e. I would've phrased this as `if !a && !b` which I think is easier to read.